rest-assured-tests/src/test/java/TariffManagementServiceClass.java

```diff
@@ -0,0 +1,39 @@
+import io.restassured.response.Response;
+
+import static io.restassured.RestAssured.*;
+import static org.junit.Assert.assertEquals;
+
+public class TariffManagementServiceClass {
+
+    static Tariff[] getTariffsList() {
+        Response response = get("/tariffs");
+        assertEquals(200, response.getStatusCode());
+        return Tariff.jsonToTariffsList(response);
+    }
+
+    static Tariff getTariffById(int id) {
+        Response response = get("/tariffs/" + id);
+        assertEquals(200, response.getStatusCode());
+        return Tariff.jsonToTariff(response);
+    }
+
+    static Response createTariffByPost(String name, float price, int washingTime, int statusCode) {
+        Response response = given()
+                .body("{\n" +
+                        "  \"name\": \"" + name + "\"," +
+                        "  \"price\": " + price + "," +
+                        "  \"washingTime\": " + washingTime + "" +
+                        "}")
+                .contentType("application/json;charset=UTF-8")
+                .post("/tariffs");
```

Comment:
too many hardcoded strings

Reply:
What do you suggest? Should I make constants instead of "application/json;charset=UTF-8" and "/tariffs"?